src/client/game/BulletPool.js

```diff
@@ -0,0 +1,29 @@
+var NUM_BULLETS = 10;
```

Comment:
Alright I like that :+1: We wouldn't want that scenario with the ark trees to happen to us